scripts/generate-static.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+## OPENAPI_FILENAME=yourapi generate.sh 
+
+# generate an API client for a service
+generate_sdk() {
+    local file_name=$1
+    local output_path=$2
+    local package_name=$3
+     
+    echo "Validating OpenAPI ${file_name}"
+    npx @openapitools/openapi-generator-cli validate -i "$file_name"
+
```

Comment:
Why the need for a separate Bash-based generation workflow for this SDK?

Reply:
Golang one is designed for autoupdates. This is more or less one time script that I will put on package readme for people to discover.

I did tried to add that to golang but that will require pre post hooks defined for each api and some small refactor so gave up